.gitignore

```diff
@@ -102,4 +102,4 @@ packages/transform-io/typescript/demo-app/
 *.egg-info
 
 native/
-
+src/docker/itk-wasm-base/itk_wasm_env_vars.sh
```

Comment:
why gitignore though? we don't expect commits often but expect local changes to the file?

Reply:
It is a derived file from ITK-Wasm/itk_wasm_env.bash. We need to add in there since Docker's build context only takes into account the local directory.